packages/react-native/src/private/webapis/performance/Performance.js

```diff
@@ -180,16 +183,9 @@ export default class Performance {
       duration = options.duration ?? duration;
     }
 
-    const measure = new PerformanceMeasure(measureName, {
-      // FIXME(T196011255): this is incorrect, as we're only assigning the
-      // start/end if they're specified as a number, but not if they're
-      // specified as previous mark names.
-      startTime,
-      duration,
-    });
-
+    let [computedStartTime, computedDuration] = [startTime, duration];
```

Comment:
I wouldn't initialize this this way. If we native module isn't defined we can fallback to nullish values.

We should clean up the defaults we're defining above as we're using 0 as default for certain things that shouldn't.

Reply:
Hmm, that is an interesting take. This would require us to change type of `PerformanceMeasure.startTime` and `duration` to include nulls, which would be a breaking change and breaking the spec. Since we are showing a warning anyway, IMO both approaches are acceptable. WDYT? 🤔